Service/ZGWService.php

```diff
@@ -27,46 +27,63 @@ public function __construct(EntityManagerInterface $entityManager, ParameterBagI
         $this->parameterBag = $parameterBag;
     }
 
-    /*
-   * Returns a welcoming string
-   *
-   * @return array
-   */
+    /** 
+     * Returns a updated zaak with a new besluit.
+     * 
+     * @param array $data 
+     * @param array $configuration 
+     *
+     * @return array Zaak.
+     */
     public function postZaakBesluitHandler(array $data, array $configuration): array
     {
         $this->data = $data;
         $this->configuration = $configuration;
+        $method = $this->data['parameters']->getMethod();
+
+        switch ($method) {
+            case 'POST':
+            case 'PUT':
+            case 'PATCH':
+                $explodedArray = explode('/api/zrc/v1/zaken/', $this->data['parameters']->getPathInfo());
```

Comment:
Did you test this against the newest version of the corebundle? In the current version, I'd expect $this->data['path'] to get this path.